opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -994,4 +998,250 @@ mod tests {
                 == AnyValue::String("Updated by FirstProcessor".into())
         );
     }
+
+    #[test]
+    fn test_simple_processor_sync_exporter_without_runtime() {
```

Comment:
nit: We can write a helper and call them for all the cominations:
1. Normal test
2. Tokio test - current thread
3. Tokio test - multi-thread
4. Tokio test - multi-thread with thread=1

we can eventually prepare a test matrix.md file for quick visualization.

Reply:
Yes, good to have a test matrix. Will do it after adding tests for batch.